### PR TITLE
Link calculate with the React components

### PR DESCRIPTION
**Pull request by uryelah**
In this milestone I:

- Linked app with the calculate function

- passed down the calculate click event down to the Button component

- passed down the result state to Display

- created a calcPath state in the App Component to be passed down to Display and show the calculation history

Thanks for the review, let me know if there are any issues or improvements I could do 🦄

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 function Button(props) {
-  const { name, color, wide } = props;
+  const {
+    name, color, wide, onClick,
+  } = props;
```
Comment (MauricioRobayo) [inline src/components/Button.js]: You could destructure on the argument: `function Button({ name, color, wide, onClick }) {`

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import Button from './Button';
 
 function ButtonPanel(props) {
-  const { styles } = props;
+  const { styles, clickHandler } = props;
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel.js]: Same here: `function ButtonPanel({ styles, clickHandler }) {`.

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -18,40 +18,41 @@ function ButtonPanel(props) {
...
+        <Button name="+" onClick={clickHandler} />
       </div>
       <div style={rowStyles}>
-        <Button wide color="grey" name="0" />
-        <Button color="grey" name="." />
-        <Button name="=" />
+        <Button wide color="grey" name="0" onClick={clickHandler} />
+        <Button color="grey" name="." onClick={clickHandler} />
+        <Button name="=" onClick={clickHandler} />
       </div>
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel.js]: If we could map through an array and build this 🤔 that would avoid so much duplicate code.

File: src/components/Display.js [hunk 4]
```diff
@@ -2,28 +2,43 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 function Display(props) {
-  const { result, styles } = props;
+  const { result, styles, top } = props;
```
Comment (MauricioRobayo) [inline src/components/Display.js]: `function Display({ result, styles, top }) {`

**Review by MauricioRobayo — APPROVED** [review]
This is it, you got it. Congratulations!

This was way easy for you, please move on to bigger challenges!

Status: __Approved__.

Optional:

- You could remove `App.test.js` and `setupTest.js` if you're not going to use them.
- The comments on the code are also optional.

File: src/components/Button.js (hunk 1 above)
Comment (uryelah) [reply src/components/Button.js]: Nice, much more cleaner

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (uryelah) [reply src/components/ButtonPanel.js]: Noted.

File: src/components/ButtonPanel.js (hunk 3 above)
Comment (uryelah) [reply src/components/ButtonPanel.js]: I'll try something about that. Great sugestion!

**Pull request merged**